src/Pact/Types/Persistence.hs

```diff
@@ -278,6 +281,8 @@ data WriteType =
 newtype TxId = TxId Word64
     deriving (Eq,Ord,Enum,Num,Real,Integral,Bounded,Default,FromJSON,ToJSON,Generic)
 
+instance Serial TxId
+instance Serialize TxId
```

Comment:
Everything looks good, but why both Serial and Serialize?

Reply:
I agree with @LindaOrtega. A quick note about the difference would be appreciated so we can reference this.